operations/addresses.md

```diff
@@ -0,0 +1,247 @@
+    "Addresses": [
+        {
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Addresses` | array of [Address add items](#address-add-items), max 1000 items | required | Collection of addresses to be created. |
+
+#### Address add items
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the [Address](configuration.md#address). |
+| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
+| `AccountType` | string | required | Discriminator specifying whether the account is a [Company](companies.md#company) or a [Customer](customers.md#customer). |
+| `Line1` | string | optional | First line of the address. |
+| `Line2` | string | optional | Second line of the address. |
+| `City` | string | optional | The city. |
+| `PostalCode` | string | optional | Postal code. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude. |
+| `Longitude` | number | optional | The longitude. |
+
+### Response
+
+```javascript
+{
+    "Addresses": [
+        {
+            "Id": "fc7b2df3-de66-48a6-907d-af4600ecd892",
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "AccountType": "Customer",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ],
+    "Cursor": null
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Addresses` | array of [Address](#address-items) | required | Created addresses. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older address items. |
+
+## Update addresses
+
+Updates an existing address in the system assigned to a specified account.
```

Comment:
You can update multiple addresses in one call, so I think it should say something like "Updates one or more existing addresses in the system, assigned to specified accounts."